sdv/metadata/single_table.py

```diff
@@ -477,6 +479,60 @@ def validate(self):
                 + '\n'.join([str(e) for e in errors])
             )
 
+    def visualize(self, show_table_details='full', output_filepath=None):
+        """Create a visualization of the multi-table dataset.
+
+        Args:
+            show_table_details (str):
+                If 'full', show the column names, primary, alternate and sequence keys are all
+                shown along with the table names. If 'summarized' primary, alternate and sequence
+                keys are shown and a count of the different sdtypes is shown. Defaults to 'full'.
+            output_filepath (str):
+                Full path of where to savve the visualization. If None, the visualization is not
+                saved. Defaults to None.
+
+        Returns:
+            ``graphviz.Digraph`` object.
+        """
+        if show_table_details not in ('full', 'summarized'):
+            raise ValueError("'show_table_details' should be 'full' or 'summarized'.")
+
+        if show_table_details == 'full':
+            columns = [
+                fr"{name} : {meta.get('sdtype')}\l"
+                for name, meta in self.columns.items()
+            ]
+            node = ''.join(columns)
```

Comment:
do you think we could pull these sections out into helper functions in the `visualize.py` file and then the multi-table and single table metadata could use them? They seem pretty similar.

Reply:
Its slightly different, in this case we always add `\l` meanwhile on multi-table we use `'\l'.join()` which skips one.